src/aat/resources/features/F-1005 - Start Case Event TTL/S-1005.5.td.json

```diff
@@ -0,0 +1,152 @@
+{
+  "_guid_": "S-1005.5",
+  "_extends_": "F-1005_RetrieveTriggerInternal",
+  "title": "must successfully update a cases TTL.SystemTTL value using V2 /cases/{caseId}/event-triggers/{eventId}",
```

Comment:
Title in scenario TD files should match feature file's corresponding scenario description.

Reply:
Will be updated by 69f4c8d387474245b16c099ef1a8b7eddf556fac in #1974.